build.sbt

```diff
@@ -1,19 +1,11 @@
-
-inThisBuild(
-  (homepage := Some(url("https://github.com/gemini-hlsw/sbt-lucuma"))) +: lucumaPublishSettings
-)
+ThisBuild / tlBaseVersion := "0.5"
+ThisBuild / scalaVersion := "2.12.15"
 
 lazy val sbtLucuma = (project in file("."))
-  .disablePlugins(LucumaPlugin)
   .enablePlugins(SbtPlugin)
-  .enablePlugins(AutomateHeaderPlugin)
   .settings(
     name := "sbt-lucuma",
-    addSbtPlugin("de.heikoseeberger"         % "sbt-header"             % "5.6.0"),
```

Comment:
Are these two included in `sbt-typelevel`?

Reply:
Yes.

Reply:
Er, to clarify: sbt-tpolecat is not included in sbt-typelevel. sbt-typelevel has its own settings plugin which is effectively the same.